### PR TITLE
Composer Packagist support

### PR DESCRIPTION
**Pull request by mbirth**
Now it's ready, too. :)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        }
+    ],
+    "require": {
+        "php": ">=5.3.0",
+        "ext-xml": "*",
+        "ext-zip": "*",
+        "seblucas/tbszip": "~2.16.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "5.4.*"
```
Comment (seblucas) [inline composer.json]: I guess this is not compatible with php 5.3.

Any version should enough for the tests, I think

**Comment by mbirth** [comment]
And there it goes green. :smile:

**Comment by seblucas** [comment]
Webhook and 1.0.0 tag done. Thanks again

**Comment by seblucas** [comment]
A small question is there any way to fix the licence on packagist while keeping the original file from  Andreas Gohr ?

**Comment by mbirth** [comment]
Of course, you need to find the best matching license from https://spdx.org/licenses/ and put that identifier in a "license" field into the composer.json.

**Pull request merged**